composer.json

```diff
@@ -26,22 +27,6 @@
       }
     }
   },
-  "exclude": [
-    "drupal/section_purger"
-  ],
-  "repositories": {
-    "drupal": {
-      "type": "composer",
-      "url": "https://packages.drupal.org/8",
-      "exclude": [
-        "drupal/section_purger"
-      ]
-    },
-    "drupal/section_purger": {
```

Comment:
no use of it here, as this will go into the project's root composer.

Reply:
@MdNadimHossain the purpose of this being here is so we can include this module in non-tide drupal projects and get all the platform stuff, including section. So we need to preserve this.

Reply:
If section_purger is being used in non-tide projects we may need to brief @nicksantamaria on the challenges we had bringing this module into D10 @MdNadimHossain.

Reply:
This has been added back.